packages/cloud-cognitive/src/components/InlineEdit/InlineEdit.js

```diff
@@ -197,9 +197,14 @@ export let InlineEdit = React.forwardRef(
     };
 
     const handleKeyDown = (ev) => {
-      if (ev.key === 'Enter') {
-        ev.preventDefault();
-        refInput.current.blur(); // will cause save
+      switch (ev.key) {
+        case 'Enter':
+          ev.preventDefault();
+          refInput.current.blur(); // will cause save
+          break;
+        case 'Escape':
+          refInput.current.blur(); // will cause save
```

Comment:
The PR title says "add cancel on use of escape key", but what you're actually doing on use of escape key is save. Just checking which of these was intended -- should escape trigger save, or should it trigger cancel? If the former, maybe fix the PR title :-) if the latter, fix the code :-)

Reply:
Ah yes, oops @dcwarwick